### PR TITLE
added new plugin sort with swaps

### PR DESCRIPTION
**Pull request by akshararama**
added the plugin sort with swaps

### CODE REVIEW

**Comment by suddenlykevin** [comment]
not money door but cool

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+            if (j < Integer.parseInt(framework.getSquare(i, 0))) {
+                return false;
+            }
+        }
+
+        return true;
+    }
+
+    public String getGameOverMessage() {
+        return "you did it in" + count + "moves!";
```
Comment (suddenlykevin) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java]: capitalization!

**Review by suddenlykevin — COMMENTED** [review]
should we capitalize messages?

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+            count = count + 1;
+        } else {
+            last = x;
+        }
+
+        this.move = !this.move;
+    }
+
+    public boolean isGameOver() {
+//        System.out.println("here");
```
Comment (dannykuo25) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java]: redundant comment

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java (hunk 1 above)
Comment (dannykuo25) [reply src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java]: interesting

**Pull request merged**